### PR TITLE
fix: export record error && dal.models is not initialized

### PR DESCRIPTION
**Pull request by Yangruipis**
## Desc

related to #422 

### fixes:

1. get right module id when export records
2. init dal.models when running import/export

### CODE REVIEW

File: pkg/envoy/store/compose.go [hunk 1]
```diff
@@ -187,7 +187,17 @@ func (d *composeDecoder) decodeComposeRecord(ctx context.Context, s store.Storer
...
-				ffNs = append(ffNs, &fNs)
+
+				nn, _, err := s.SearchComposeModules(ctx, types.ModuleFilter{
+					NamespaceID: nsID,
+				})
+				if err != nil {
+					continue
+				}
+				for _, n := range nn {
+					fNs.ModuleID = n.ID
```
Comment (tjerman) [inline pkg/envoy/store/compose.go:197]: Won't this only include the last module in the slice? Shouldn't we copy the `fNs`?

**Review by tjerman — CHANGES_REQUESTED** [review]

File: pkg/envoy/store/compose.go (hunk 1 above)
Comment (Yangruipis) [reply pkg/envoy/store/compose.go:197]: yes you are right, it is fixed now

**Comment by Yangruipis** [comment]
@tjerman this pr is updated, plz take a look, thanks

**Comment by tjerman** [comment]
Lgtm

**Pull request merged**